docker/Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM python:alpine as production
+
+# Configuration params
+ARG INVENTREE_REPO="https://github.com/inventree/InvenTree.git"
+ARG INVENTREE_VERSION="master"
+ARG INVENTREE_HOME="/home/inventree"
+
+ENV PYTHONUNBUFFERED 1
+
+# InvenTree paths
+ENV INVENTREE_SRC_DIR="$INVENTREE_HOME/src"
+ENV INVENTREE_STATIC_ROOT="$INVENTREE_HOME/static"
+ENV INVENTREE_MEDIA_ROOT="$INVENTREE_HOME/media"
+ENV INVENTREE_LOG_DIR="$INVENTREE_HOME/log"
+ENV INVENTREE_BACKUP_DIR="$INVENTREE_HOME/backup"
+ENV INVENTREE_VENV="$INVENTREE_HOME/env"
+
+LABEL org.label-schema.schema-version="1.0" \
+      org.label-schema.build-date=$DATE \
+      org.label-schema.vendor="inventree" \
+      org.label-schema.name="inventree/inventree" \
+      org.label-schema.url="https://hub.docker.com/r/inventree/inventree-docker" \
+      org.label-schema.version=$INVENTREE_VERSION \
+      org.label-schema.vcs-url=$INVENTREE_REPO \
+      org.label-schema.vcs-branch=$BRANCH \
+      org.label-schema.vcs-ref=$COMMIT
+
+RUN echo "Installing InvenTree '${INVENTREE_VERSION}' from ${INVENTREE_REPO}"
+
+# Create user account
+RUN addgroup -S inventreegroup && adduser -S inventree -G inventreegroup
```

Comment:
If I run a docker container as myself (1000, 1000) and have a volume for the config files.  And then if the docker container has inventree user/group set to 500,500 (e.g.).  Then my home directory outside of docker will contain files with owner 500,500.   

The linuxserver.io series of docker containers support PUID, PGID as integers for process uid and process groupid.  

Usually I set these to be 1000,1000 because that's my user id from which I run docker containers.

ARG INVENTREE_GID=1000
ARG INVENTREE_UID=1000

RUN addgroup -S inventree --gid ${INVENTREE_GID} && adduser -S inventree -G inventree --uid ${INVENTREE_UID}